docs/docs/rest-api/public/api/v2/types/stringTypes.raml

```diff
@@ -22,13 +22,14 @@ types:
       validation has historically been lax.
   PathId:
     type: string
-    pattern: ^(\/?((\.\.)|(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9]))?($|\/))+$
+    pattern: ^(\/?((\.\.)|(([a-z0-9]|[a-z0-9][a-z0-9\-_]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-_]*[a-z0-9]))?($|\/))+$
```

Comment:
What are we using to validate containers? Cause `_` should not be allowed for container names. https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/core/task/Task.scala#L306

Reply:
We should add a unit test to make sure we never allow `_` in container names